docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
       SUPERSET_USERNAME: ''
       SUPERSET_PASSWORD: ''
       PUSH_NOTIFICATION: 'enable'
+      EXPERIMENTAL_FORM_DATA: 'enable'
```

Comment:
Please don't add env vars to docker compose for options that are not required. It fills the docker compose file with options that are not relevant to most developers and obscures options which are important.